lib/index.js

```diff
@@ -112,8 +159,47 @@ MaplibreGeocoder.prototype = {
         "</div></div>"
       );
     },
+    processResults: function (searchResults, suggestionResults, limit) {
+      if (!suggestionResults || suggestionResults.length === 0) {
+        return searchResults;
+      }
+
+      // Filter out the suggestions that are already in the search results
+      const featureLabels = searchResults.map((feature) => feature.place_name);
```

Comment:
This library unfortunately doesn't have an es6 transpilation step so for now we have to use es5 syntax i.e no arrow functions

Reply:
Updated to ES5 functions